omim2obo/main.py

```diff
@@ -220,7 +220,7 @@ def omim2obo(use_cache: bool = False):
             graph.add((axiom, OWL.annotatedSource, omim_uri))
             graph.add((axiom, OWL.annotatedProperty, oboInOwl.hasExactSynonym))
             graph.add((axiom, OWL.annotatedTarget, Literal(abbreviation)))
-            graph.add((axiom, OBOINOWL.hasSynonymType, MONDONS.abbreviation))
+            graph.add((axiom, OBOINOWL.hasSynonymType, OMO['0003000']))
```

Comment:
`OBOINOWL` --> `oboInOwl`

Thanks for changing to the property `OMO_0003000`.

Reply:
Hmm I didn't think I'd made that change here yet. I'll change it back.

Reply:
Oh! I didn't actually change that in here.

I remember now; there were duplicate namespaces; one `oboInOwl` and another `OBOINOWL`. Anywhere there's only the former now.